excellent/scanner.go

```diff
@@ -37,6 +37,7 @@ type xscanner struct {
 	unreadRunes         []rune
 	unreadCount         int
 	identifierTopLevels []string
+	unescapeBody        bool // unescape @@ sequences in the body
```

Comment:
so that when we're using this for expression migrating, we can leave the `@@` sequences in the body